diesel_tests/tests/expressions/ops.rs

```diff
@@ -91,6 +91,6 @@ fn mix_and_match_all_numeric_ops() {
         (3, 'Jim'), (4, 'Bob')").unwrap();
 
     let expected_data = vec![4, 6, 7, 9];
-    let data = users.select(id * 3 / 2 + 4 - 1).load(&connection);
+    let data = users.select(id * 3 / 2 + 4 - 1).order(id).load(&connection);
```

Comment:
As a general rule of thumb we try to keep things small, focused, and easy to review. For example, #508, #509, and #510 could have reasonably been a single PR, but it helps the reviewer a ton to keep things more granular.